persistent/Database/Persist/Quasi/Internal.hs

```diff
@@ -349,6 +352,9 @@ toParsedEntityDef lwc = ParsedEntityDef
     (attribs, extras) =
         parseEntityFields fieldLines
 
+    schemaName =
+      fmap SchemaNameDB $ listToMaybe $ mapMaybe (T.stripPrefix "schema=") entAttribs
```

Comment:
Depending on what packages are available, `Data.List.Extra.firstJust` does this nicely